src/ui/LiveSplit.tsx

```diff
@@ -61,6 +61,10 @@ export class LiveSplit extends React.Component<{}, State> {
     constructor(props: {}) {
         super(props);
 
+        if ("serviceWorker" in navigator) {
+            navigator.serviceWorker.register("/service-worker.js");
+        }
+
```

Comment:
This is slightly suboptimal as we don't install the service worker in a non-production build, so this 404s in a local environment, as it should. But it still looks bad in the debugger.

Reply:
We can add `process.env.NODE_ENV === 'production'` to the `if` condition.